sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileGetRangeListDiffOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Optional parameters for Get Range List Diff.
+    /// </summary>
+    public class ShareFileGetRangeListDiffOptions : ShareFileGetRangeListOptions
```

Comment:
We decided in Java that if we made an options bag for ShareFileGetRangeList that we would keep them entirely separate in case one day (knowing our luck) a param exists for only one of the APIs?

Reply:
@kasobol-msft what is your thoughts on this one?

Reply:
+1